### PR TITLE
Draft Response to Reviewers

### PR DESCRIPTION
**Pull request by cgreene**
Initial steps in this area. Currently WIP. @agitter feel free to pick up if you get time.

### CODE REVIEW

**Comment by cgreene** [comment]
Ok - @agitter : this is off to you! Please note that I left two elements of the table for you to fill in. :)

**Comment by agitter** [comment]
@cgreene I made a few more changes and think this is just about ready.  Let's proofread one more time and merge it (and submit!).

File: content/02.intro.md [hunk 1]
```diff
@@ -55,28 +55,29 @@ We sketch out a few simple example architectures in Figure @fig:nn-petting-zoo.
...
-| Adversarial training | A process by which artificial training examples are maliciously designed to fool a NN and then input as training examples to make the resulting NN robust (no relation to GANs) |
-| Data augmentation | A process by which transformations that do not affect relevant properties of the input data (e.g., arbitrary rotations of histopathology images) are applied to training examples to increase the size of the training set. |
+Table @tbl:glossary also provides select example applications, though in practice each neural network architecture has been broadly applied across multiple types of biomedical data.
+A recent book from Goodfellow et al. covers neural network architectures in detail [@url:http://www.deeplearningbook.org/], and LeCun et al. provides a more general introduction [@doi:10.1038/nature14539].
```
Comment (cgreene) [inline content/02.intro.md]: `provide` instead of `provides`? I think it's based on the many authors as opposed to the single entity paper.

**Comment by cgreene** [comment]
Ok - with my change this is 👍 ! I think we are ready to go. Let me know & send over the marked document. I can submit tomorrow morning. 😀

**Pull request merged**